webpagesish/data/addon-groups.js

```diff
@@ -0,0 +1,99 @@
+export default [
+  // Only popup:
+  {
+    id: "recentlyUsed",
+    name: () => {
+      chrome.i18n.getMessage("recentlyUsed");
+    },
```

Comment:
probably a mistake, there is no return here

Reply:
that is in fact a mistake